unit_tests/test_aggregator.py

```diff
@@ -0,0 +1,415 @@
+# pylint: disable=missing-function-docstring
+from unittest.mock import patch, PropertyMock
+import pytest
+
+from gsy_e_sdk.aggregator import Aggregator, AggregatorWebsocketMessageReceiver
+from unit_tests.constants import (test_simulation,
+                                  test_agg, test_batch_command_dict,
+                                  test_response, TEST_AGGREGATOR_PREFIX,
+                                  TEST_CONFIGURATION_PREFIX)
+
+
+#####################################
+#            FIXTURES               #
+
+@pytest.fixture(autouse=True)
+def mock_connections(mocker):
+    """Fixture mocks methods and functions which establish external connections"""
+    mocker.patch("gsy_e_sdk.aggregator.blocking_get_request",
+                 return_value=test_agg, autospec=True)
+    mocker.patch("gsy_e_sdk.aggregator.blocking_post_request",
+                 return_value=test_agg, autospec=True)
+    mocker.patch("gsy_e_sdk.clients.rest_asset_client.retrieve_jwt_key_from_server",
+                 return_value="some_key")
+    mocker.patch("gsy_framework.client_connections.utils.RestCommunicationMixin"
+                 + "._create_jwt_refresh_timer")
+    mocker.patch("gsy_e_sdk.aggregator.Aggregator._connect_to_simulation",
+                 autospec=True)
+
+
+@pytest.fixture(autouse=True)
+def mock_prefixes(mocker):
+    """
+    Fixture mocks external prefix calculation functions
+    """
```

Comment:
Nit:
```suggestion
    """Fixture mocks external prefix calculation functions."""
```
Also, docstrings should usually use verbs in the imperative form to describe what the function does. It's probably not always necessary, but we use it as our rule most of the time. Please take some time to check our guidelines about it :)